backend/src/main/java/com/elshop/backend/common/AppUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.elshop.backend.common;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.springframework.stereotype.Component;
+
+import java.util.UUID;
+
+@Component
+public class AppUtils {
```

Comment:
Der Klassenname ist nicht sehr aussagekräftig. Vorschlag:
```suggestion
public class RandomIdService {
```

Reply:
PS: oder eben `RandomKeyService`